pykusto/_src/functions.py

```diff
@@ -1130,14 +1133,20 @@ def any_if(expr: ExpressionType, predicate: BooleanType) -> _AnyAggregationExpre
     def arg_max(*args: ExpressionType) -> _AnyAggregationExpression:
         """
         https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/arg-max-aggfunction
+        to use wildcard simply put only one column.
         """
+        if len(args) == 1:
+            return _AnyAggregationExpression(KQL(f"arg_max({args[0]}, *)"))
```

Comment:
How about we introduce `expr_to_maximize` argument before `*args`? It will enforce at least one arg and make this code look more like the `any(...)` above.

Reply:
sure I can do that :)